hr_employee_ppe/README.rst

```diff
@@ -37,26 +34,13 @@ Installation
 
 To install this module, you need to have HR module installed or it will be requested during installation.
 
-Usage
-=====
-
-To manage PPEs allocation to employees, you will have to populate a list of equipments first.
-
-1. Go to menu *Employees -> PPEs -> Equipments*
-
-Then you can allocate:
-
-1. Go to menu *Employees -> Employees*
-2. Select an employee
-3. Go to *PPE* tab
-
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues <https://github.com/OCA/hr/issues>`_.
+Bugs are tracked on `GitHub Issues <https://github.com/tegin/hr/issues>`_.
```

Comment:
Why not OCA ?

Reply:
You are right, it should be OCA. Fixed :wink: